lib/chef/api_client.rb

```diff
@@ -1,7 +1,7 @@
 #
-# Author:: Adam Jacob (<adam@chef.io>)
-# Author:: Nuo Yan (<nuo@chef.io>)
-# Copyright:: Copyright (c) 2008, 2015 Chef Software, Inc.
+# Author:: Adam Jacob (<adam@opscode.com>)
+# Author:: Nuo Yan (<nuo@opscode.com>)
+# Copyright:: Copyright (c) 2008 Opscode, Inc.
```

Comment:
It was a side effect of reverting the commit which updated that.

We should probably do a mass update of the copyright headers to prevent this kind of noise.